UCSD-E4E/Pyrenote
# Fix label newline Bug

There was a bug where when uploading via file, newlines were preserved in the output. Update will strip newlines ('\n' and '\r' from both uploaded labels and downloaded CVSs

- to test:
   -  Upload labels via csv file
   - Make an annotation using those new labels
   - observe successful output of data in csv format

Closes #251
